tox.ini

```diff
@@ -16,7 +16,7 @@ deps=
     jsonschema
     osqp~=0.6 ; python_version < "3.10"
     pytest
-    pytest-benchmark
+    pytest-benchmark~=4.0.0
```

Comment:
I'm guessing you mean >= 4.0.0 and <5. The way you wrote it means >=4.0.0 and <4.1 though.

```suggestion
    pytest-benchmark~=4.0
```

Reply:
I will pull those in from the other PR. Thx!